tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -591,24 +591,97 @@ test.describe( 'Set up accounts', () => {
 				).toBeEnabled();
 			} );
 
-			test( 'should display the correct Google Ads ID when connected', async () => {
-				const googleAccountCard =
-					setUpAccountsPage.getGoogleAccountCard();
+			test( 'should send an API request to connect existing Google Ads account', async () => {
 				const googleAdsAccountCard =
 					setUpAccountsPage.getGoogleAdsAccountCard();
 
 				const once = setUpAccountsPage.fulfillTimes( 1 );
-				await once.fulfillAdsAccounts( [ { id: 12345 } ] );
-				await once.mockAdsAccountConnected();
+
 				await once.mockAdsStatusClaimed();
+				await once.fulfillAdsAccounts( {
+					id: 111111,
+				} );
```

Comment:
The payload should be `Array<Object>` type.

Reply:
👍